docs/_documentations/intellij-getting-started.md

```diff
@@ -12,35 +12,17 @@ type: document
 Use Codewind on IntelliJ to develop your applications.
 
 ## Installing Codewind on IntelliJ
-Complete the prerequisites and choose to install Codewind on IntelliJ with either the custom plug-in repository or the Eclipse Downloads page.
+Complete the prerequisites and install Codewind on IntelliJ.
 
 ### Prerequisites
 - Install [IntelliJ](https://www.jetbrains.com/idea/download/#section=mac).
 - Install Docker.
 - If you use Linux, also install Docker Compose.
 
-### Installing Codewind on IntelliJ with the custom plug-in repository
-1. From IntelliJ, open either **Preferences** or **Settings**. A new window appears.
-   - On macOS, go to **Main**>**Preferences**.
-   - On Windows, go to **File**>**Settings**.
-2. Click **Plugins**.
-3. Click the **Manage Repositories, Configure Proxy or Install Plugin from Disk** gear icon.
-4. From the menu, click the **Manage Plugin Repositories...** gear icon.
-5. Click the **+** button.
-6. Enter `https://download.eclipse.org/codewind/milestone/0.9.0/updatePlugins.xml`.
-7. **For IntelliJ on Linux and Windows:** After you paste the URL, click outside the entry field so that the URL is deselected. If the URL isn't deselected before clicking **OK**, the URL is saved as a blank entry, and an error occurs.
-8. Click **OK**.
-9. Click the **Marketplace** heading.
-10. In the search field, type `codewind`. The `codewind-intellij` plug-in appears. Click **Install**.
-
-### Installing Codewind on IntelliJ from the Eclipse Downloads page
-1. Download the [codewind-intellij-0.9.0.zip](https://download.eclipse.org/codewind/milestone/0.9.0/codewind-intellij-0.9.0.zip) file to your machine.
-2. From IntelliJ, open either **Preferences** or **Settings**. A new window appears.
-   - On macOS, go to **Main**>**Preferences**.
-   - On Windows, go to **File**>**Settings**.
-3. Click **Plugins**.
-4. Click the **Manage Repositories, Configure Proxy or Install Plugin from Disk** gear icon. From the menu, click **Install Plugin from Disk...**. A window appears that displays the files on your computer.
-5. Choose the `codewind-intellij-0.9.0.zip` file and click **Open**.
+### Installing
+1. Go to the Codewind page on the [JetBrains website](https://plugins.jetbrains.com/plugin/13839-codewind).
```

Comment:
These instructions are for downloading the plugin zip from the JetBrains marketplace.  We should rather tell the users how to install the plugin from within the IDE.

1. From IntelliJ, open either **Preferences** or **Settings**. A new window appears.
   - On macOS, go to **Main**>**Preferences**.
   - On Windows or Linux, go to **File**>**Settings**.
2. Click **Plugins**.
3. Clike the **Marketplace** tab
4. In the search field, type `codewind`.  The `Codewind` plugin appears.  Click **Install**